dotnet-statsig-tests/Server/DedupeTest.cs

```diff
@@ -0,0 +1,201 @@
+using System.Collections.Generic;
+using Xunit;
+using WireMock.Server;
+using WireMock.RequestBuilders;
+using WireMock.ResponseBuilders;
+using System.Threading.Tasks;
+using WireMock;
+using WireMock.Settings;
+using Statsig;
+using Statsig.Server;
+using Newtonsoft.Json.Linq;
+using WireMock.ResponseProviders;
+
+namespace dotnet_statsig_tests
+{
+    [Collection("Statsig Singleton Tests")]
```

Comment:
If you are you the driver directly, I don't think you need this decorator.

Reply:
Will remove.